README.md

```diff
@@ -160,15 +160,27 @@ error, the `geolocationError` callback is passed a
 
  This plugins requires the following usage description:
 
- * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location. 
+ * `NSLocationWhenInUseUsageDescription` describes the reason that the app accesses the user's location, this used while the app is running in the foreground.
+ * `NSLocationAlwaysAndWhenInUseUsageDescription` describes the reason that the app is requesting access to the user’s location information at all times, use this key if your iOS app accesses location information while running in the background and foreground. 
+ * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times, use this key if your iOS app accesses location information in the background, and you deploy to a target earlier than iOS 11. In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.
```

Comment:
```suggestion
 * `NSLocationAlwaysUsageDescription` describes the reason that the app is requesting access to the user's location at all times. Use this key if your iOS app accesses location information in the background. and you deploy to a target earlier than iOS 11. In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.
```

> In that case, add both this key and `NSLocationAlwaysAndWhenInUseUsageDescription` to your app’s `Info.plist` file with the same message.

Can you clarify this? Are we suppose to use `NSLocationAlwaysAndWhenInUseUsageDescription` on iOS >= 12?

Reply:
Hello,
Yes as apple mentioned in this link: https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationalwaysandwheninuseusagedescription